app/components/works/edit/directory_component.rb

```diff
@@ -9,7 +9,11 @@ def initialize(directory:, form:)
         @form = form
       end
 
-      delegate :name, :children_directories, :children_files, :depth, to: :directory
+      delegate :name, :children_directories, :children_files, :depth, :full_path, to: :directory
+
+      def dir_id
+        "dir-#{directory.index}"
+      end
 
```

Comment:
any ideas why it claims that line is untested?  it seems like any render of the work edit page (like the new cypress test) should it?